packages/s2-core/src/facet/base-facet.ts

```diff
@@ -304,6 +306,14 @@ export abstract class BaseFacet {
     }
   };
 
+  onSelectAll = () => {
```

Comment:
可以放到  `RootInteraction` 里面, 用户通过 spredsheet.interaction.selectAllCells() 实现全选

```suggestion
  selectAllCells = () => {
      this.spreadsheet.interaction.changeState({
        stateName: InteractionStateName.ALL_SELECTED,
      });
  };
```